[2021-11-29 12:34:59,853] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_1 2020-02-26T00:00:00+00:00 [queued]>
[2021-11-29 12:35:00,247] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: parallel_dag.task_1 2020-02-26T00:00:00+00:00 [queued]>
[2021-11-29 12:35:00,247] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-11-29 12:35:00,247] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-11-29 12:35:00,247] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-11-29 12:35:00,535] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): task_1> on 2020-02-26T00:00:00+00:00
[2021-11-29 12:35:00,695] {standard_task_runner.py:52} INFO - Started process 153746 to run task
[2021-11-29 12:35:00,859] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'parallel_dag', 'task_1', '2020-02-26T00:00:00+00:00', '--job-id', '387', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmp88205cj7', '--error-file', '/tmp/tmpp0suqis2']
[2021-11-29 12:35:00,860] {standard_task_runner.py:77} INFO - Job 387: Subtask task_1
[2021-11-29 12:35:03,001] {logging_mixin.py:104} INFO - Running <TaskInstance: parallel_dag.task_1 2020-02-26T00:00:00+00:00 [running]> on host airflowvm
[2021-11-29 12:35:04,374] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=task_1
AIRFLOW_CTX_EXECUTION_DATE=2020-02-26T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-02-26T00:00:00+00:00
[2021-11-29 12:35:04,377] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-11-29 12:35:04,434] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'sleep 3']
[2021-11-29 12:35:04,522] {subprocess.py:75} INFO - Output:
[2021-11-29 12:35:07,757] {subprocess.py:83} INFO - Command exited with return code 0
[2021-11-29 12:35:08,520] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=task_1, execution_date=20200226T000000, start_date=20211129T123459, end_date=20211129T123508
[2021-11-29 12:35:09,507] {taskinstance.py:1245} INFO - 2 downstream tasks scheduled from follow-on schedule check
[2021-11-29 12:35:09,515] {local_task_job.py:151} INFO - Task exited with return code 0
